### PR TITLE
Add Edge Chromium and interest links.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updating the String for Edge Chromium

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Solving issue #3816

File: WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-Browser-WIA.md [hunk 1]
```diff
@@ -56,5 +56,10 @@ MSIE 10.0; Windows NT 6|IE 10.0 for Windows XP and newer versions of desktop ope
 Windows NT 6.3; Trident/7.0</br></br>Windows NT 6.3; Win64; x64; Trident/7.0</br></br>Windows NT 6.3; WOW64; Trident/7.0| Windows 8.1 desktop operating system, different platforms|
 Windows NT 6.2; Trident/7.0</br></br>Windows NT 6.2; Win64; x64; Trident/7.0</br></br>Windows NT 6.2; WOW64; Trident/7.0|Windows 8 desktop operating system, different platforms|
 Windows NT 6.1; Trident/7.0</br></br>Windows NT 6.1; Win64; x64; Trident/7.0</br></br>Windows NT 6.1; WOW64; Trident/7.0|Windows 7 desktop operating system, different platforms|
+Edg/79.0.309.43 | Microsoft Edge (Chromium) | 
 MSIPC| Microsoft Information Protection and Control Client|
 Windows Rights Management Client|Windows Rights Management Client|
+
+#AdditionalLinks
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-Browser-WIA.md]: Looks like you might have trouble with your Space Bar key...? 😉 Also: 1 single hashtag is only used for page titles.
```suggestion
### Additional links
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  I did not realize it :P Thank you again! :)

**Comment by illfated** [comment]
@JohanFreelancer9 : Feel free to review & comment, in case there are any other interesting formatting details I may have missed in this document page.

**Comment by illfated** [comment]
#sign-off

**Comment by eross-msft** [comment]
@billmath Please confirm if this text is okay to merge. Thanks!!

**Comment by billmath** [comment]
#sign-off

**Comment by j0rt3g4** [comment]
@kenwith  @scanum can you take ownership of this one?

**Pull request merged**